modules/central-configuration-options-operator.adoc

```diff
@@ -287,7 +287,7 @@ The default value is `scanner-v4-db`.
 | Configures a monitoring endpoint for Scanner V4. The monitoring endpoint allows other services to collect metrics from Scanner V4, provided in a Prometheus-compatible format. Use `Enabled` to expose the monitoring endpoint. When you enable monitoring, {product-title-short} creates a new service, `monitoring`, with port 9090, and a network policy allowing inbound connections to the port. By default, this is not enabled.
 
 | `scannerV4.scannerComponent`
-| Enables Scanner V4. The default value is `default`, which is disabled. To enable Scanner V4, set this parameter to `Enabled`.
+| If this setting is not specified, by default during installation, Scanner V4 is enabled for new installation. For updates from an earlier release, if Scanner V4 was not enabled, it will not be enabled in the upgrade by default. Disabling Scanner V4 negatively impacts the ability of {product-title-scan} to provide thorough and accurate scanning results.
```

Comment:
nit: just an idea

```suggestion
| If this setting is not specified, Scanner V4 is enabled for new installations, by default. For updates from an earlier release, it will not be enabled in the upgrade by default, if Scanner V4 was not previously enabled. Disabling Scanner V4 negatively impacts the ability of {product-title-scan} to provide thorough and accurate scanning results.
```